ibis/omniscidb/client.py

```diff
@@ -488,6 +488,51 @@ def alter(self, tbl_properties=None):
         """
         raise NotImplementedError('This method is not implemented yet!')
 
+    def insert(
+        self,
+        obj: Union[pd.DataFrame, ir.Expr],
```

Comment:
pls use the same api and in impala / spark

Reply:
Ok, fixed. I have tried to align API as much as possible.